XNAControls/XNAListBoxItem.cs

```diff
@@ -76,22 +72,17 @@ public string Text
     public int TextYPadding { get; set; }
 
     public int TextXPadding { get; set; }
+
     public bool Selectable { get; set; } = true;
 
-    private float alpha = 0.0f;
+    private float alpha;
+
     public float Alpha
     {
-        get { return alpha; }
-        set
-        {
-            if (value < 0.0f)
-                alpha = 0.0f;
-            else if (value > 1.0f)
-                alpha = 1.0f;
-            else
-                alpha = value;
-        }
+        get => alpha;
+
+        set => alpha = value < 0.0f ? 0.0f : value > 1.0f ? 1.0f : value;
```

Comment:
I prefer the original over chained ternary operators, I find them hard to read at a glance.

This comment also applies to other chained ternary operators here.